### PR TITLE
eslint upgrade

### PR DESCRIPTION
**Pull request by snownoop**
@ButuzGOL please continue

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -43,12 +43,14 @@
     "classnames": "^2.2.5",
     "moment": "^2.13.0",
     "svg4everybody": "^2.0.3",
-    "uuid": "^2.0.2"
+    "uuid": "^2.0.2",
+    "react": "^15.0.2",
```
Comment (stomita) [inline package.json]: You shouldn't include the react in dependencies of main package.json, which will cause the duplicate install.

File: src/scripts/Checkbox.js [hunk 2]
```diff
@@ -38,8 +40,6 @@ export default class Checkbox extends Component {
 }
 
 Checkbox.propTypes = {
-  className: PropTypes.string,
-  label: PropTypes.string,
```
Comment (stomita) [inline src/scripts/Checkbox.js]: These props are actually used inside of component, even if no lint error.

File: examples/client/scripts/components/DatepickerExamples.js [hunk 3]
```diff
@@ -56,6 +57,12 @@ export default class DatepickerExamples extends Component {
             />
           </div>
         </div>
+        <h2 className='slds-m-vertical--medium'>DateInput</h2>
+        <div style={ styles }>
+          <div style={ { width: '20rem' } }>
+            <DateInput/>
```
Comment (stomita) [inline examples/client/scripts/components/DatepickerExamples.js]: You need DateInput example in DatepickerExample ? DateInput examples are already in FormExample.

**Comment by ButuzGOL** [comment]
@snownoop @stomita this pull moved to #105

**Pull request closed without merging**